### PR TITLE
Overall improvements

### PR DESCRIPTION
**Pull request by lukybil**
- removed mocked tickets
- now using the normalized zone in the `MapZoneBottomSheet`
- purchase screen context exclusive to purchase screen, a nested stack navigator had to be used
- prefetch for some queries, can be easily extended
- prepared `useStaticArcgisData` for fetching from a static CDN, URLs must be updated
- grouped query options to one file

### CODE REVIEW

File: components/special/OmnipresentComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { usePrefetchOnAppStart } from '@/modules/backend/hooks/usePrefetchOnAppStart'
+
+const OmnipresentComponent = () => {
+  usePrefetchOnAppStart()
```
Comment (radoslavzeman) [inline components/special/OmnipresentComponent.tsx:5]: Why this one-time hook? :D

File: modules/backend/hooks/usePrefetchOnAppStart.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      await Promise.all(
+        emails.data.verifiedEmails.map((emailDto) =>
+          queryClient.prefetchQuery(parkingCardsOptions({ email: emailDto.email })),
+        ),
+      )
+  }, [queryClient])
+
+  useEffect(() => {
+    InteractionManager.runAfterInteractions(() => prefetch())
+  }, [prefetch])
```
Comment (radoslavzeman) [inline modules/backend/hooks/usePrefetchOnAppStart.tsx:31]: This whole prefetch + OmnipresentComponent asks for some explanation in comment, or at least a link to docs.

File: modules/arcgis/hooks/useArcgisData.ts [hunk 3]
```diff
@@ -49,16 +50,13 @@ export const useArcgisData = () => {
...
-    rawOdpData: rawOdpData as FeatureCollection<Polygon, GeoJsonProperties>,
-    rawZonesData: rawZonesData as FeatureCollection<Polygon, GeoJsonProperties>,
+    rawAssistantsData: rawAssistantsData as FeatureCollection<Point, AssistantPoint> | undefined,
+    rawParkomatsData: rawParkomatsData as FeatureCollection<Point, ParkomatPoint> | undefined,
+    rawPartnersData: rawPartnersData as FeatureCollection<Point, PartnerPoint> | undefined,
+    rawParkingLotsData: rawParkingLotsData as FeatureCollection<Point, ParkingLotPoint> | undefined,
+    rawBranchesData: rawBranchesData as FeatureCollection<Point, BranchPoint> | undefined,
+    rawUdrData: rawUdrData as FeatureCollection<Polygon, MapUdrZone> | undefined,
+    rawOdpData: rawOdpData as FeatureCollection<Polygon, GeoJsonProperties> | undefined,
+    rawZonesData: rawZonesData as FeatureCollection<Polygon, GeoJsonProperties> | undefined,
```
Comment (radoslavzeman) [inline modules/arcgis/hooks/useArcgisData.ts:60]: I'm nervous about this whole custom typing :D

**Review by radoslavzeman — APPROVED** [review]

File: modules/arcgis/hooks/useArcgisData.ts (hunk 3 above)
Comment (lukybil) [reply modules/arcgis/hooks/useArcgisData.ts:60]: We will not be using that hook anymore anyway

**Pull request merged**